### ISSUE TITLE
flowblades speed up slow down motion problem

### ISSUE THREAD

**Issue by houku** [jliljebl/flowblade#890]
so, just to point out, i did NOT come from adobe, and i WANT to use flowblade, but my editing involves speeding up the video and sometimes slowing down, now i KNOW theres a feature for that, but i dont like the fact i need to wait for my entire video to be sped up so i can show a montage, i know i probably need to change my workflow if the feature stays like this, but i would like to be able to speed up the video without making a separate file, kinda like you do in most editors, if that will happen, i will gladly move to flowblade 100%

thanks (thats my only problem at the moment)

**Comment by jliljebl** [comment]
>  but i dont like the fact i need to wait for my entire video to be sped up so i can show a montage

Yes if you have a monatage on timeline you probably need to render that out before doing speedup/slow.

Make sure that you are aware of the fact that you can render only parts of mediaitems by the following workflow:

1) Open media
2) Double click to open media item in clip monitor
3) Set In/out points around area you want to slowdown/speed up
4) Select "Render Slow/Fast..." option with right click from media item
5) Select option "Form Mark In/to Mark Out" for option render range.

**Comment by jose1711** [comment]
I second this request. Even better would be if speed is a key-able attribute as in Blender VSE for instance or in http://slowmovideo.granjow.net/docs.html which would allow for smooth speed transitions.

**Comment by jliljebl** [comment]
This functionality has not been improved in quite some time so I think it is time to start looking into this again. I'm thinking 2.10 could see some work here, 2.8 seems not likely currently. The keyframed speed change editor seems doable, not sure if we will be using curves for this, depends on quite a few factors.

**Comment by jliljebl** [comment]
Now that we have timeline rendering this feature makes some sense because it will possible to do realtime playback on those clips. We could eventually get around to doing this but probably not on next cycle.

**Comment by jliljebl** [comment]
Closing as `wontfix`. We will be employing rendered motion clips going forward and there are no current plans to change this.

Rendered motion clips take sometime to create but are more performant. It is now also possible to render motion clips on timeline in addition rendering new media items, so the workflow has been improved a lot since this issue was created.
